app/scripts/modules/appengine/domain/IAppengineLoadBalancer.ts

```diff
@@ -1,6 +1,7 @@
 import {ILoadBalancer} from 'core/domain/index';
 
 export interface IAppengineLoadBalancer extends ILoadBalancer {
+  credentials?: string;
   split: IAppengineTrafficSplit;
   migrateTraffic: boolean;
```

Comment:
I'd add a warning banner in the wizard if the server group setup doesn't match the setup required for `migrateTraffic`.